azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeCapabilitiesChangeNotificationMode.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.calling.models;
+
+import com.azure.android.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/**
+ * Represents notification mode for user capabilities changed.
+ */
+public final class CallCompositeCapabilitiesChangeNotificationMode
```

Comment:
Is this part of the Public API that we are going to have?

Reply:
Yes